client/src/components/TicketForm/TicketForm.jsx

```diff
@@ -85,7 +85,7 @@ class TicketForm extends Component {
 
 		if (this.state.description.value && this.state.location.value && this.state.contact.value) {
 			this.props.submitTicket({
-				requestorId: 'dummyUserID',	// TODO: fetch the user ID
+				requestorId: '5af12d0e2e439d0010e8d3e4',	// TODO: fetch the user ID
```

Comment:
Question: what is this? Should this be committed to the repo for everyone?

Reply:
I think Dustin's PR lets you access the currently logged in user + their ID in the redux store.

Reply:
@TimothyGu  it was my dummy account's User id.
@shannonphu ok will look into it